src/NServiceBus.Core/Routing/Legacy/ConfigureMSMQDistributor.cs

```diff
@@ -20,6 +21,7 @@ public static void EnlistWithLegacyMSMQDistributor(this EndpointConfiguration co
             config.GetSettings().Set("LegacyDistributor.Address", masterNodeAddress);
             config.GetSettings().Set("LegacyDistributor.ControlAddress", masterNodeControlAddress);
             config.GetSettings().Set("LegacyDistributor.Capacity", capacity);
+            config.DisableFeature<TimeoutManager>();
```

Comment:
since we only use the distributors timeout manager if the messages origins from the distributor, I don't think should disable the local TM.